samcli/local/docker/container.py

```diff
@@ -200,7 +199,7 @@ def wait_for_logs(self, stdout=None, stderr=None):
         real_container = self.docker_client.containers.get(self.id)
 
         # Fetch both stdout and stderr streams from Docker as a single iterator.
-        logs_itr = attach(self.docker_client, container=real_container, stdout=True, stderr=True, logs=True)
+        logs_itr = real_container.attach(stream=True, logs=False, demux=True)
```

Comment:
Is this with the latest version of docker? do we need any version bumps?

Reply:
This was released in 3.7 (looking at https://github.com/docker/docker-py/issues/1952). We are on 4.0 or greater, so we should be covered and not need any bump.